src/Zicht/Bundle/AdminBundle/Command/DumpRoleHierarchyCommand.php

```diff
@@ -138,7 +139,7 @@ function ($n) {
         ];
 
         $output->writeln('# Generated by command ' . $this->getName() . ' (' . __CLASS__ . ')');
-        $output->writeln('# ' . join(' ', array_map('escapeshellarg', $_SERVER['argv'])));
+        $output->writeln('# ' . str_replace('\'', '', join(' ', array_map('escapeshellarg', $_SERVER['argv']))));
```

Comment:
Ik weet niet of dit nog ergens op slaat. Blijkbaar is `escapeshellarg` niet erg intelligent en wrapt ie gewoon alles in single quotes i.p.v. te detecteren of escaping nodig is. Dan kun je nu net zo goed die `array_map('escapeshellarg', ...)` weghalen en gewoon direct `join(' ', $_SERVER['argv'])` doen.